### DISCUSSION TITLE
[Feature] Install via APK. When the version is updated, it can connect to the server to update the APK online.

### DISCUSSION THREAD

**Discussion by qinggeng2015** [immich-app/immich#13694] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Currently, Immich is installed via Google Playstore, but the version updates on the Playstore are often delayed, causing the client version to lag behind the service version. If using the APK installation method, updating the APK is not particularly convenient. Could it be possible to implement a feature where, upon launching the APK, it requests the server's interface to check for version updates? If a new version is available, the corresponding APK can be downloaded from the server for an online upgrade. If there are no significant changes between versions, the upgrade would be optional.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by mmomjian** [comment]
It’s unlikely that we would spend the time developing this. There are methods for updating apps from other repos like f droid.

**Reply by qinggeng2015** [reply]
I see that the latest version of the Immich app on F-Droid is still 1.118.1, not 1.118.2, so it cannot guarantee complete consistency with the server version.

**Reply by mmomjian** [reply]
I think there’s other ways, but regardless we will not be able to add this to immich. Thanks for understanding

**Reply by qinggeng2015** [reply]
Alright, if you have a better solution, I hope you can share it. Thank you!

**Reply by TheDidek** [reply]
Builds are published on the repo's releases tab. I can recommend [Obtainium](https://github.com/ImranR98/Obtainium) that can get and auto-update APKs from GitHub.

**Reply by qinggeng2015** [reply]
Thank you so much! It looks like it can meet the requirements. I'll give it a try!

**Reply by NicholasFlamy** [reply]
I created #15779 as a feature request that would make it easy to synchronize your server and app version with Obtainium.
